pkg/render/logstorage/linseed/linseed_test.go

```diff
@@ -77,6 +77,7 @@ var _ = Describe("Linseed rendering tests", func() {
 		}
 
 		BeforeEach(func() {
+			replicas = 2
```

Comment:
Is this change needed? I'm concerned that it might have been nil before for a reason (but not obvious why)